functions/filter/‏‏ex3.js

```diff
@@ -0,0 +1 @@
+const filterPastDates = (arrOfDatesStr) => arrOfDatesStr.filter(dateStr => new Date > new Date(dateStr))
```

Comment:
Instead of creating  a new date every iteration create a new date at the start of the function and use it in each iteration of the filter